From a6bdbf5a52c1fa5c4d0286b9472c50a3ee7d1272 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Thu, 22 Aug 2013 17:37:30 +0800
Subject: [PATCH 0097/1074] ENGR00276335 net:fec: fix build warning

drivers/net/ethernet/freescale/fec_main.c:864
warning: 'vlan_tag' may be used uninitialized in this function
note: 'vlan_tag' was declared here

Compiler with the warning option: -Wuninitialized, the warning
will be reported.

Signed-off-by: Fugang Duan  <B38611@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 drivers/net/ethernet/freescale/fec_main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/freescale/fec_main.c b/drivers/net/ethernet/freescale/fec_main.c
index 602f794..807192d 100644
--- a/drivers/net/ethernet/freescale/fec_main.c
+++ b/drivers/net/ethernet/freescale/fec_main.c
@@ -866,7 +866,7 @@ fec_enet_rx(struct net_device *ndev, int budget)
 	int	pkt_received = 0;
 	struct	bufdesc_ex *ebdp = NULL;
 	bool	vlan_packet_rcvd = false;
-	u16	vlan_tag;
+	u16	vlan_tag = 0;
 	int	index = 0;
 
 #ifdef CONFIG_M532x
-- 
1.7.5.4

